### PR TITLE
New scripts for updating people.md

### PR DESCRIPTION
**Pull request by thibautjombart**
I will put guidelines on how to use these in the readme after the PR goes through. But **do not use for now** as some members are still not in the membership page I am pulling data from, and would be erased by the update.

The current implementation works as follows:

1. start R into reconhub.github.io/R 
2. type: `source("update_people.R")
3. type `update_people_file()` to generate a new `people.md` whose 'member' section will be built from the registration form; note that photos will need adding in the approrpiate plate; to add photos by default, use `update_people_file(add_missing_pic = TRUE)`

See doc of the function for more info:

```
#' This function generates a new, updated people.md
#'
#' The function will read the current people.md file, import membership data,
#' generate entries for all members in the registration spreadsheet, and insert
#' these new data in the 'people-list' section in a new, updated people.md
#' file. Note that if unsure, you can specify an alternative output file, so
#' that you can compare the old and new version to make sure nothing got
#' lost. In particular, make sure all members are recorded in the registration
#' spreadsheet, as only these will be present in the updated version.
#'
#' @param in_file the file to be used as input, defaults to `people.md`
#'
#' @param out_file the file to be used as output, defaults to the same as
#'   `in_file` in which case the input file will be replaced by the new version.
#' 
#' @param add_missing_pic a logical indicating if a default 'anonymous' pic
#'   should be created to replace missing photos
```

## Left to do

* handle non-ascii characters in names (it is a total faff for file names)
* document this in the readme
* test on new membership data once all members have been added

### CODE REVIEW

**Comment by thibautjombart** [comment]
Ascii characters are now handled too. Also added doc to README

File: R/update_people.R [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+  out <- c(out, img_txt)
+
+  ## if image does not exist, copy the anonymous pic by default
+  path_to_pic <- sub("^[ ]*img: ", "", img_txt)
+
+  ## handle non ascii characters like ö
+  path_to_pic <- gsub("&ouml;", "o", path_to_pic)
+
+  if (!file.exists(path_to_pic)) {
```
Comment (TimTaylor) [inline R/update_people.R:94]: @thibautjombart I think you want
``` 
!if (file.exists(paste0("..", path_to_pic)))
```

**Review by TimTaylor — COMMENTED** [review]
one issue with checking for existing picture but apart from that it LGTM

File: R/update_people.R (hunk 1 above)
Comment (thibautjombart) [reply R/update_people.R:94]: Oh, and that line 96 was an ad hoc fix which is no longer needed. Thanks

Comment (TimTaylor) [reply R/update_people.R:94]: yep - wasn't quite sure as saw you were using stringi for the ascii but figured it wouldn't hurt :)

**Pull request merged**